recipes/bakta/meta.yaml

```diff
@@ -1,16 +1,16 @@
-{% set version = "1.5.1" %}
+{% set version = "1.6.0" %}
 
 package:
   name: bakta
   version: '{{ version }}'
 
 source:
   url: https://github.com/oschwengers/bakta/archive/v{{ version }}.tar.gz
-  sha256: 'f7bef81e569e16d0b261e54eddef659129b7edee0d0929c87385136a0f292eea'
+  sha256: 'b752a891a75bbc6af69110c71a42e9b83bdf385770ac3359eeba5465915743fc'
 
 build:
   noarch: python
-  number: 0
+  number: 1
```

Comment:
@oschwengers you can reset this to 0 when you release a new version

Reply:
thanks